g/gsl/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,327 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.2",
+	"packages": {
+		"gsl": {
+			"build-depends": [
+				"pkgconfig(openblas)"
+			],
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libm.so.6(x86_64))",
+				"soname(libopenblas.so.0(x86_64))"
+			],
+			"files": [
+				"/usr/bin/gsl-config",
```

Comment:
`/usr/bin/gsl-config` and `/usr/share/man/man1/gsl-config.1` should be moved into devel.